pkgs/racket-doc/syntax/scribblings/modcode.scrbl

```diff
@@ -7,23 +7,26 @@
 
 @defproc[(get-module-code [path path-string?]
                           [#:submodule-path submodule-path (listof symbol?) '()]
-                          [#:sub-path compiled-subdir0 (and/c path-string? relative-path?) (default-compiled-sub-path)]
-                          [compiled-subdir (and/c path-string? relative-path?) compiled-subdir0]
+                          [#:sub-path compiled-subdir0
+                           (or/c (and/c path-string? relative-path?)
+                                 (listof (and/c path-string? relative-path?)))
+                           (default-compiled-sub-path)]
```

Comment:
Is this meant to change to `(use-compiled-file-paths)`?

Reply:
Right! This also triggered a number of other smaller changes to the docs. I'll push after it builds and I look at it.